source/retryable-reads/tests/aggregate-serverErrors.yml

```diff
@@ -1,3 +1,6 @@
+runOn:
+    - { minServerVersion: "4.0" }
```

Comment:
Technically all the files that use `failCommand` (all of them I believe) should be skipped on 4.0 sharded clusters so this should be:
```
runOn:
    - { minServerVersion: "4.0", topology: ["single", "replicaset"]}
    - { minServerVersion: "4.1.8" }

Reply:
Just missed this while merging :man_facepalming: 

I'll fix it in a follow-up PR.

Reply:
https://github.com/mongodb/specifications/pull/525